Automattic/wp-calypso
# A4A > Marketplace: Add tab content header for all the hosting tabs

## Proposed Changes

* This PR adds the content header for all the hosting tabs

## Why are these changes being made?

* For the new hosting page

## Testing Instructions

* Open the A4A live link
* Go to Marketplace - Hosting: Verify you can see the content header for all the tabs as shown below:

<img width="1728" alt="Screenshot 2024-07-29 at 10 48 04 AM" src="https://github.com/user-attachments/assets/42f6896d-ef59-43b1-ae95-d8acdde1d6c7">
<img width="1728" alt="Screenshot 2024-07-29 at 10 48 10 AM" src="https://github.com/user-attachments/assets/fa0e0d61-548c-4d4f-8a69-ed67b8b920af">
<img width="1728" alt="Screenshot 2024-07-29 at 10 48 17 AM" src="https://github.com/user-attachments/assets/7400dd52-7649-4c54-b9e9-555a8eee0afe">


## Pre-merge Checklist
